packages/darklang/languageTools/writtenTypesToProgramTypes.dark

```diff
@@ -1416,5 +1416,7 @@ module Darklang =
         : (ProgramTypes.SourceFile.SourceFile *
           List<Parser.Range * RuntimeErrors.NameResolution.Error>)
         =
+        let pm = LanguageTools.PackageManager.pm ()
```

Comment:
Should pm be sent as a parameter?

Reply:
Eventually, yes. I suspect we had just punted that, as it'll require us to trickle the `pm` object in a bunch of places.